lib/azure/AzureDataStore.ts

```diff
@@ -27,7 +27,7 @@ export type AzureDSInputOptions = {
 /**
  * Azure data store class that implements IDataStore
  */
-export default class AzureDataStore {
+export default class AzureDataStore extends iDatastore.Adapter {
```

Comment:
Not your fault, can we do at the top:
```javascript
const { Adapter, Key, Errors } = require('interface-datastore');
```
So we can get rid of the `iDatastore` ugliness?

```suggestion
export default class AzureDataStore extends Adapter {
```